### PR TITLE
Add config fields for txFiller minGasPrice and maxGasPrice

### PR DESCRIPTION
**Pull request by yondonfu**
This PR adds the `minGasPrice` and `maxGasPrice` fields in the `blockchain` object of a config file. The test-harness will parse these fields if they are set and pass the values into the transaction filler service.

### CODE REVIEW

File: src/networkcreator.js [hunk 1]
```diff
@@ -1117,6 +1117,18 @@ class NetworkCreator extends EventEmitter {
...
+      if (this.config.blockchain.minGasPrice) {
+        txFillerService.command.push('-minGasPrice')
+        txFillerService.command.push(this.config.blockchain.minGasPrice)
+      }
+      if (this.config.blockchain.maxGasPrice) {
+        txFillerService.command.push('-maxGasPrice')
+        txFillerService.command.push(this.config.blockchain.maxGasPrice)
+      }
+    }
+
```
Comment (kyriediculous) [inline src/networkcreator.js:1127]: This works ! Just some nits here and there that could make it a little bit shorter. 

We could omit the `this.hasGeth` check since that's being checked in the return statement's ternary expression.

To then check whether an object key exists in an object we won't know will be defined we can do `(object.foo||{}).bar` and not run into `cannot access field bar of undefined`  

Javascript's `Array.push` just like go's append can take multiple arguments to append to the array so we could shorten this to: 

```
    if ((this.config.blockchain||{}).minGasPrice) {
      txFillerService.command.push('-minGasPrice', this.config.blockchain.minGasPrice)
    }
    if ((this.config.blockchain||{}).maxGasPrice) {
      txFillerService.command.push('-maxGasPrice', this.config.blockchain.maxGasPrice)
    }
```

**Review by kyriediculous — CHANGES_REQUESTED** [review]

File: src/networkcreator.js (hunk 1 above)
Comment (yondonfu) [reply src/networkcreator.js:1127]: Thanks for the tip!

Fixed in 82311d4

**Review by kyriediculous — COMMENTED** [review]
LGTM 🚀  , needs a rebase !

**Comment by yondonfu** [comment]
Bump - needs a follow up review.

**Review by kyriediculous — APPROVED** [review]
LGTM 🚀

**Review by ya7ya — APPROVED** [review]
